testsuite/MDAnalysisTests/coordinates/test_copying.py

```diff
@@ -37,19 +37,25 @@
     DMS,
```

Comment:
🤦🏽 I had completely forgot this set of tests existed (to be fair, looks like we've been forgetting about it for the last few readers too).

Reply:
You earned the test archaeologist badge 🏅